index.html

```diff
@@ -7,7 +7,24 @@
     <link rel="stylesheet" href="style.css" />
   </head>
   <body>
-    <div class="container"></div>
+    <div class="container">
+      <div class="container-list">
+        <h3 class="title pending">
+          대기중 (<span class="count count-pending">0</span>)
+        </h3>
+        <ul class="list list-pending">
+        </ul>
+        <h3 class="title done">
+          완료됨 (<span class="count count-done">0</span>)
+        </h3>
+        <ul class="list list-done">
+        </ul>
+      </div>
+      <form class="form container-form">
+        <input class="input-text" type="text" placeholder="새로운 할일 추가">
+        <input class="input-button" type="button" value="+">
+      </form>
+    </div>
```

Comment:
CSS 클래스 네이밍 규칙을 통일하는 것이 좋습니다.

지금은 `.title.pending`, `.list.list-pending`, `.form.container-form`이 세 가지 방식이 혼용되고 있죠?
정답은 없지만, `.[상위개념 클래스].[상위개념 클래스]-[하위개념 클래스]`로 작성하는 패턴이 일반적입니다.
이를 고려했을 때, 코드를 다음과 같이 고쳐볼 수 있습니다.

```suggestion
    <div class="container">
      <div class="container-list">
        <h3 class="title title-pending">
          대기중 (<span class="count count-pending">0</span>)
        </h3>
        <ul class="list list-pending">
        </ul>
        <h3 class="title title-done">
          완료됨 (<span class="count count-done">0</span>)
        </h3>
        <ul class="list list-done">
        </ul>
      </div>
      <form class="form container-form">
        <input class="input-text" type="text" placeholder="새로운 할일 추가">
        <input class="input-button" type="button" value="+">
      </form>
    </div>
```

Reply:
```<h3 class="title title-done">``` 이러한 방식으로 네이밍 하는것이 권장되나요?
"title-done"로도 충분하다고 느껴지는데, 이유가 무엇인지 궁금합니다.

Reply:
`[상위개념 클래스] [상위개념 클래스]-[하위개념 클래스]` 메모... 감사합니다!! 이런 규칙이 정한다면 다음에 구성을 할 때 더 편할 거 같네요 👍

Reply:
어떤 방식을 사용하든 전혀 상관 없습니다! 다만 재용님이 작성한 방식에 맞게 추천해드린 거죠 ㅎㅎ

.title-pending만으로 적는 것보다
.title.title-pending, .title.title-done으로 나누어 적으면 공통 부분은(.title) 클래스를 지정해서 스타일링 하고, 더 세부적인 부분은 .title.title-pending 을 통해서 스타일링하여 코드의 중복을 줄일 수 있다는 장점이 있습니다.

특히 이런 패턴은 utility 중심의 UI Framework(bootstrap, semantic ui, bulma) 등에서 많이 사용됩니다